src/streamlink_cli/constants.py

```diff
@@ -13,7 +13,8 @@
 SUPPORTED_PLAYERS = {  # these are the players that streamlink knows how to set the window title for with `--title`. key names are used in help text
     # name: possible binary names (linux/mac and windows)
     "vlc": ["vlc", "vlc.exe"],
-    "mpv": ["mpv", "mpv.exe"]
+    "mpv": ["mpv", "mpv.exe"],
+    "pot": ["potplayer", "potplayermini64.exe", "potplayermini.exe"]
```

Comment:
Potplayer is always used as Potplayer

I can't find any references with just the name `pot`
so it should be named `potplayer` instead of `pot`

Reply:
Renamed